c++ std compiler flags and gnu-extensions

Hi,
I'm trying to link against spirv-cross but get various errors:
undefined reference to `std::__cxx11::basic_string

I believe it's because SPIRV-Cross is compiled using:
set(CMAKE_CXX_STANDARD 11)
Internally, cmake uses the flag "-std=gnu++11" and enables gnu extensions.  

It might be better to also set:
set(CMAKE_CXX_EXTENSIONS OFF)
unless SPIRV-Cross uses gnu-specific extensions.  

This might simplify the build for other programs that link against SPIRV-Cross?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,6 +21,7 @@
 
 cmake_minimum_required(VERSION 3.0)
 set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_EXTENSIONS OFF)
 
 # Avoid a warning if parent project sets VERSION in project().
 if (${CMAKE_VERSION} VERSION_GREATER "3.0.1")

